examples/SS316L.json

```diff
@@ -129,6 +129,18 @@
             "reference": "None",
             "uncertainty": "None",
             "print_symbol": "\\alpha"
+        },
+        "interface_response_function": {
+            "print_name": "Dendrite tip velocity",
+            "value_laurent_poly": [[0,0],[-0.000517783154178829,1],[0.000231067291419276, 2]],
```

Comment:
Would it help streamline this workflow if ExaCA was modified to read this json file directly, instead of requiring its own special format? I don't think it would be an extremely heavy lift or anything, though I would want to think about how to handle the situation where multiple interfacial response functions are present (i.e., a ferritic and austenitic phase)

Reply:
Maybe? But I don't think that every code will ever adopt this format so we'll end up doing conversions in Myna no matter what. And a small json conversion dictionary is easy enough

Reply:
I think @stvdwtt has a way to assign phase-specific properties. Let's plan on implementing that in the future when multi-phase CA gets implemented?